peps/pep-0772.rst

```diff
@@ -0,0 +1,463 @@
+will facilitate:
+
+* Work with PSF staff and the newly formed `User Success Workgroup`_ toward
+  improving the user experience around packaging.
+* A more stable packaging ecosystem that is more responsive to community input.
+* Increased transparency and clearer shared goals for the packaging ecosystem.
+* Increased tactical and fundraising support from the PSF, to increase capacity
+  and funding available to packaging tools.
+
+=============
+Specification
+=============
+
+-----------------
+Packaging Council
+-----------------
+
+The Packaging Council will be composed of 5 individuals.
+
+Mandate
+=======
+
+The council shall work to:
+
+* Improve Python packaging's user experience.
+* Maintain the quality and stability of the Python packaging standards.
+* Formalise and maintain the relationship with the core team as well as the
+  PSF.
+* Establish appropriate decision-making processes.
+* Seek consensus among contributors before acting in a formal capacity.
+* Make contributing as accessible, inclusive, and sustainable as possible.
+
+Responsibilities
+================
+
+The council shall:
+
+* have broad authority over the Python packaging standards and Python Packaging
+  User Guide, that are maintained on packaging.python.org.
+* establish processes for making binding decisions regarding packaging
+  standards, tools and implementations as well as for considering
+  ecosystem-wide changes.
+* look for ways to use their direct authority as little as possible – seeking
+  consensus/consent over voting.
+
+To use its powers, the council votes. Every council member must either vote or
+explicitly abstain. Members with conflicts of interest on a particular vote
+must abstain. Passing requires support from a majority of non-abstaining
+council members.
+
+Whenever possible, the council is expected to share its decisions and processes
+publicly in a timely fashion.
+
+Delegations
+-----------
+
+The Python Steering Council would be expected to delegate decision making to
+the Packaging Council for PEPs related to the Python packaging.  The bodies
+would work together on issues that intersect the packaging domain and language
+stewardship (including the CPython implementation, standard library, and
+distribution).
+
+The PSF Board would be encouraged to formally deprecate the Packaging Workgroup
+and the Packaging Council would wholly take on the responsibilities of the
+PSF's Packaging Workgroup.
+
+The PyPA would be expected to work with the Packaging Council to establish a
+decision making process that governs the technical projects under the PyPA
+umbrella.
+
+Processes
+=========
+
+Election of the council
+-----------------------
+
+A council election consists of two phases:
+
+* Phase 1: A member of the Packaging community (defined later in this document)
+  can self-nominate themselves for the council elections.
+* Phase 2: Each core team member can assign zero to five stars to each
```

Comment:
"Core team member" needs more explicit definition or reference to its definition. Is there a published list of core team members now?

Reply:
I think the private https://github.com/python/voters repo has the definitive list (119 members). There's also the public https://devguide.python.org/core-developers/developer-log/ (119 members) which might not quite align, and https://github.com/orgs/python/teams/python-core/members (131 members) which I'm not sure is public.

The annual SC election PEPs at https://peps.python.org/topic/governance/ list the active team members for each election (last time there were 100 https://peps.python.org/pep-8106/#complete-voter-roll) and is generated from https://github.com/python/voters.

Reply:
Actually, I think "core team" here is referring to "PyPA community membership", and this is what @willingc says needs clarifying (and as my confusion shows!)

We can reference the PEP 13 section further down in:

> * Interested core team members: Any Python core team member who is willing to
  participate is welcome.

Reply:
Yes, I would like clarification of Packaging core team. I'm not sure who that includes.

Reply:
FWIW, this was a copy-paste mistake -- this has now been amended to correctly state "Packaging community member" instead of "core team member".